dotcom-rendering/src/model/buildLightboxImages.ts

```diff
@@ -31,6 +31,31 @@ const isLightboxable = (image: EnhancedImageForLightbox): boolean => {
 	}
 };
 
+/**
+ * Older legacy images use a different url format so we need to use a different
+ * approach when extracting the id of the image
+ */
+const decideImageId = (image: EnhancedImageForLightbox): string | undefined => {
+	const firstImageUrl = image.media.allImages[0]?.url;
+	if (!firstImageUrl) return;
+	const url = new URL(firstImageUrl);
+	switch (url.hostname) {
+		case 'media.guim.co.uk': {
+			// E.g.
+			// https://media.guim.co.uk/be634f340e477a975c7352f289c4353105ba9e67/288_121_3702_2221/140.jpg
+			// This bit                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+			return url.pathname.split('/')[1];
+		}
+		case 'static.guim.co.uk':
+		// E.g.
+		// https://static.guim.co.uk/sys-images/Guardian/Pix/pictures/2015/5/26/1432666797165/59de49e2-553f-4b52-b7ac-09bda7f63e4b-220x132.jpeg
+		// This bit                                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+		default: {
+			return url.pathname.split('/').reverse()[0];
```

Comment:
We’ve favoured [`Array.prototype.at`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at) with negative values elsewhere:

```suggestion
			return url.pathname.split('/').at(-1);
```